File: daily/27. 移除元素.cpp
class Solution {
public:
    int removeElement(vector<int>& nums, int val) {
        int sum=0;
        for(int i=0;i<nums.size();i++){
            if(nums[i]!=val){
                nums[sum++]=nums[i];
            }
        }
        return sum;
    }
};

File: daily/双指针/双指针easy/27. 移除元素.js
/**
 * @param {number[]} nums
 * @param {number} val
 * @return {number}
 */
var removeElement = function(nums, val) {
    var sum=0;
        for(let i=0;i<nums.length;i++)
            if(nums[i]!=val)
                nums[sum++]=nums[i];
    return sum;
};